smart-contracts/contracts/interfaces/IPublicLock.sol

```diff
@@ -125,6 +125,13 @@ interface IPublicLock
    */
   function numberOfOwners() external view returns (uint);
 
+  /**
+  * A function which returns a subset of the keys for this Lock as an array
+  * @param _page the page of key owners requested when faceted by page size
+  * @param _pageSize the number of Key Owners requested per page
+  */
+  function getOwnersByPage(uint _page, uint _pageSize) external view returns (address[] memory);
```

Comment:
Hum. I am not sure I understand why we need this?

Reply:
yep removed